src/components/CommonForm/commonForm.tsx

```diff
@@ -11,6 +11,13 @@ const useStyles = makeStyles((theme: Theme) =>
 			color: theme.palette.background.paper,
 			marginRight: theme.spacing(2),
 		},
+		cancelButton: {
+			marginRight: theme.spacing(2),
+			padding: theme.spacing(1),
+			"&:hover": {
+				color: "#d32f2f !important",
```

Comment:
Please use colors from the theme object provided by the material ui only. If you wish to add a new color , do that in the theme object

Reply:
updated?